tools/submission/submission-checker.py

```diff
@@ -1630,33 +1661,43 @@ def check_compliance_acc_dir(test_dir):
       diff = files_diff(list_files(test_acc_path), REQUIRED_TEST01_ACC_FILES_1 if acc_passed else REQUIRED_TEST01_ACC_FILES)
       if diff:
         log.info("%s has file list mismatch (%s)", test_acc_path, diff)
+        is_valid = False
 
   return is_valid
 
-def check_compliance_dir(compliance_dir, model, scenario):
+def check_compliance_dir(compliance_dir, model, scenario, config):
   compliance_perf_pass = True
+  compliance_perf_dir_pass = True
   compliance_acc_pass = True
-  test_list = ["TEST01", "TEST04-A", "TEST04-B", "TEST05"]
+  test_list = ["TEST01", "TEST04", "TEST05"]
 
-  if model in ["rnnt", "bert-99", "bert-99.9", "dlrm-99", "dlrm-99.9", "3d-unet-99", "3d-unet-99.9"]:
-    test_list.remove("TEST04-A")
-    test_list.remove("TEST04-B")
+#  if model in ["rnnt", "bert-99", "bert-99.9", "dlrm-99", "dlrm-99.9", "3d-unet-99", "3d-unet-99.9"]:
```

Comment:
Why is this commented out? Still seems valid?

Reply:
Removed it for some debugging and forgot to change it back.